README.md

```diff
@@ -7,6 +7,43 @@ This repository contains reusable GitHub Workflows shared across the Flux contro
 
 ## Workflows
 
+### Release Flux controllers
+
+The [controller-release](.github/workflows/controller-release.yaml) workflow automates the release of
+Flux controllers by building and publishing container images to GitHub Container Registry (GHCR) and Docker Hub,
+and creating a GitHub release with the changelog.
+
+Example usage:
+
+```yaml
+name: release
+
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'image tag prefix'
+        default: 'rc'
+        required: false
+
+jobs:
+  release:
+    permissions:
+      contents: write # for creating the GitHub release.
+      id-token: write # for creating OIDC tokens for signing.
+      packages: write # for pushing and signing container images.
+    uses: fluxcd/gha-workflows/.github/workflows/controller-release.yaml@vX.Y.Z
+    with:
+      controller: ${{ github.event.repository.name }}
+      release-candidate-prefix: ${{ github.event.inputs.tag }}
+    secrets:
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+      dockerhub-token: ${{ secrets.DOCKER_FLUXCD_PASSWORD }}
+```
+
```

Comment:
Please list here all the 3rd-party actions used, these are the actions which don't come from GitHub nor from the fluxcd org.

Also list the outputs.